make-wheels.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
```

Comment:
I don't personally have any major objections to this, but I'm wondering if maybe we should keep the project pure python2 (for a better developer experience) for now.